docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,834 @@
+
+To do this, call the `WithLifetime` method and pass in `ContainerLifetime.Persistent`:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var queue = builder.AddRabbitMQ("rabbit")
+                   .WithLifetime(ContainerLifetime.Persistent);
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(queue).WaitFor(queue);
+
+builder.Build().Run();
+```
+
+The dashboard shows persistent containers with a pin icon:
+
+:::image type="content" source="media/persistent-container.png" alt-text="Persistent containers":::
+
+After the app host has stopped, the container will continue to run:
+
+:::image type="content" source="media/persisent-container-dockerdesktop.png" alt-text="Docker desktop showing rabbit mq":::
+
+The container persistence mechanism attempts to identify when you may wish to recreate the container. For example if the environment
+for the container changes then the container will be restarted so that you don't need to manually stop the container if you have changed
+the input configuration for the resource.
+
+### Resource commands
+
+The app host supports adding custom commands to resources. This is useful when you want to run custom commands on a resource. This can be done by calling the `WithCommand` method and passing in the command to run:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var cache = builder.AddRedis("cache")
+                   .WithClearCommand();
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(cache)
+       .WaitFor(cache);
+
+builder.Build().Run();
+```
+
+The command implementation:
+
+```csharp
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using StackExchange.Redis;
+
+public static class RedisCommandExtensions
+{
+    public static IResourceBuilder<RedisResource> WithClearCommand(
+        this IResourceBuilder<RedisResource> builder)
+    {
+        builder.WithCommand(
+            "clear-cache",
+            "Clear Cache",
+            async context =>
+            {
+                var redisConnectionString = await builder.Resource.GetConnectionStringAsync() ??
+                    throw new InvalidOperationException(
+                        "Unable to get the Redis connection string.");
+                
+                using var connection = ConnectionMultiplexer.Connect(redisConnectionString);
+
+                await connection.GetDatabase().ExecuteAsync("FLUSHALL");
+
+                return CommandResults.Success();
+            },
+            context =>
+            {
+                if (context.ResourceSnapshot.HealthStatus is HealthStatus.Healthy)
+                {
+                    return ResourceCommandState.Enabled;
+                }
+
+                return ResourceCommandState.Disabled;
+            });
+        return builder;
+    }
+}
+```
+
+These commands can be run from the dashboard:
```

Comment:
Do we need to call out that these commands are only available during local development and not when you use the dashboard in ACA?